### PR TITLE
Mantis: https://mantis.fortinet.com/bug_view_page.php?bug_id=0871038

### PR DESCRIPTION
**Pull request by cs-ankit-gupta**
Description:
Correlating Hunt with Created Indicator and Create Alert.

UTC:
1. Verified that Hunt gets correlated with created indicator and created alerts.

### CODE REVIEW

File: info.json [hunk 1]
```diff
@@ -21,17 +21,17 @@
         }
     ],
     "prerequisite": "",
-    "certified": "false",
```
Comment (cs-suyog-jinturkar) [inline info.json:24]: why this is removed?

**Review by cs-suyog-jinturkar — COMMENTED** [review]
Added one review comment

File: info.json (hunk 1 above)
Comment (cs-ankit-gupta) [reply info.json:24]: by giving certified value as false it is showing Certified as Yes
Screenshot 2023-03-01 at 4 46 27 PM

Comment (cs-suyog-jinturkar) [reply info.json:24]: Please file a mantis for this on platform UI

**Review by cs-suyog-jinturkar — APPROVED** [review]

**Pull request merged**